ci-operator/config/openshift/cincinnati/OWNERS

```diff
@@ -2,6 +2,6 @@
 # This file just uses aliases defined in OWNERS_ALIASES.
 
 approvers:
-  - installer-approvers
+  - approvers-approvers
```

Comment:
`cincinnati-approvers`?

Reply:
How did you guess that? :-D
Thanks for catching of course. Good reminder of not doing things in a rush.